pkg/security/securitycontextconstraints/provider.go

```diff
@@ -269,6 +269,28 @@ func (s *simpleProvider) ValidatePodSecurityContext(pod *api.Pod, fldPath *field
 
 	allErrs = append(allErrs, s.sysctlsStrategy.Validate(pod)...)
 
+	if len(pod.Spec.Volumes) > 0 && len(s.scc.AllowedFlexVolumes) > 0 && sccutil.SCCAllowsFSType(s.scc, securityapi.FSTypeFlexVolume) {
```

Comment:
I don't want to modify this PR, could we merge it when tests will be green and postpone this refactoring a bit?